### PR TITLE
[iOS] Fix SmallCard component layout

### PR DESCRIPTION
**Pull request by ostk0069**
## Issue
- close #ISSUE_NUMBER

## Overview (Required)
- fix https://github.com/DroidKaigi/conference-app-2021/pull/543#pullrequestreview-702295416  issue
  - `SmallCard` component seems broken by using `GeometryReader` so fixed by using constant values
  - fix other components(`LargeCard`, `MediumCard`, etc) to follow the way `SmallCard` fixed.

## Links
-

## Screenshot
Before | After
:--: | :--:
 |

### CODE REVIEW

File: ios/Sources/Component/Card/SmallCard.swift [hunk 1]
```diff
@@ -34,127 +36,96 @@ public struct SmallCard: View {
...
+                    Spacer(minLength: 4)
 
-                        Text(date.formatted)
-                            .font(.caption)
-                            .foregroundColor(AssetColor.Base.tertiary.color)
-                    }
+                    Text(date.formatted)
+                        .font(.caption)
+                        .foregroundColor(AssetColor.Base.tertiary.color)
+                }
```
Comment (ry-itto) [inline ios/Sources/Component/Card/SmallCard.swift:61]: Why is this wrapped with Group?

Comment (ostk0069) [reply ios/Sources/Component/Card/SmallCard.swift:61]: Current code was `VStack` with spacing zero. If there's any spacing `Group` is minimum view builder.
Also if I remove Group from this code, spacing 12 will added to each texts.

**Comment by ostk0069** [comment]
@ry-itto 
fixed! 🙇

**Review by ry-itto — APPROVED** [review]
LGTM ! Thanks 🙇🏼

**Pull request merged**